### ISSUE TITLE
capacitor.config.js is ignored

### ISSUE THREAD

**Issue by iamso** [HausennTechnologies/capacitor-set-version#381]
Hi

When I run `npx capacitor-set-version ./ -v 1.0.0 -b 1`, I get the following error message:
```
CustomError: Invalid CapacitorJS project: neither capacitor.config.ts nor capacitor.config.ts exist in folder ./
```
I have a `capacitor.config.js` file in the root of the project, but it seems to be ignored.
From the error message, it looks the script is trying to locate the same file twice.

Is this a bug or have you removed support for `capacitor.config.js`?

Thanks
Steve

**Comment by iamso** [comment]
Added pull request https://github.com/HausennTechnologies/capacitor-set-version/pull/382

**Comment by DKrepsky** [comment]
closed by #382
